packages/generate-persisted-query-manifest/README.md

```diff
@@ -96,6 +96,27 @@ Default:
 ]
 ```
 
+#### Usage with GraphQL Codegen persisted documents
```

Comment:
Hmm. I feel like the docs could somehow be a bit clearer that there are two distinct advantages to taking this approach:

- The sorta obvious reason: if you've already carefully configured graphql-codegen to know exactly which files contain GraphQL documents and you trust it to have done so accurately, you might as well just piggy-back on its work rather than having to configure our tool identically.
- The less obvious reason: if you're using certain graphql-codegen features (like the client-preset) then the actual documented being passed to Apollo Client is not the value you wrote in your source code, but the outcome of some light transforms applied by graphql-codegen.  While those transforms are intended to be similar to the ones that AC does, they aren't quite the same, so to analyze things accurately we should be running our tool against the output of that process.

I'm not really sure how to reword that in a clearer and more succinct manner but it doesn't seem like the README right now tries to express this?  Though maybe it's subtle enough to be futile.

Reply:
Ya thats a good point. I'll take a stab at conveying both and see how it feels.

Reply:
Took a stab at both with https://github.com/apollographql/apollo-utils/pull/414/commits/b16fcf2ea89f643821c1a4fdc80d777d5d29c7f4. Let me know what you think!